tests/installation/scc_registration.pm

```diff
@@ -21,7 +21,10 @@ use registration;
 
 sub run() {
     if (!get_var("HDD_SCC_REGISTERED")) {
-        assert_screen("scc-registration", 100);
+        if (!check_screen 'scc-registration', 210) {
+            record_soft_failure 'bsc#990254';
+            assert_screen('scc-registration', 100);
+        }
```

Comment:
of course pass and soft fail = pass
fail = fail
so at the end we have two cases :P